alphazero/_game.py

```diff
@@ -195,9 +195,10 @@ def board_after_move(board, player: int, move: (int, int)):
                 f"invalid move {move}: space already occupied in board {board}"
             )
 
-        board[move] = player
+        new_board = board.clone()
+        new_board[move] = player
 
-        return board
+        return new_board
```

Comment:
is there a reason for returning a copy of the board instead of the in-place change? I'm guessing this is somehow interfering with your MCTS. If so that's fine, I just want to make sure :)

Reply:
yeah it's mutating the state I put in which is not what I want :P

Reply:
Cool, just making sure.